.github/workflows/release.yml

```diff
@@ -118,6 +125,29 @@ jobs:
             $(printf -- "--tag ${{ vars.DOCKER_REPO }}:%s " ${tags[@]}) \
             $(printf "${{ vars.DOCKER_REPO }}@%s " ${digests[@]})
           echo "::endgroup::"
+          # Output digests as a JSON array for use in a matrix
+          echo "digests=$(printf '%s\n' "${digests[@]}" | jq -R . | jq -s .)" >> $GITHUB_OUTPUT
+    outputs:
+      digests: ${{ steps.publish-docker-hub.outputs.digests }}
+
+  attest-docker-images:
+    name: Publish Docker images attestations
+    if: startsWith(github.ref, 'refs/tags/v') || github.ref == 'refs/heads/main'
+    runs-on: ubuntu-latest
+    needs: docker-release
+    permissions:
+      id-token: write
```

Comment:
I think this specific permission isn't necessary

Reply:
But the docs say so: https://github.com/actions/attest?tab=readme-ov-file#usage

Reply:
My bad!